lsp/core/aws-lsp-core/src/http/requesters.ts

```diff
@@ -24,5 +24,5 @@ export class DefaultHttpRequester implements HttpRequester {
 }
 
 export function getETag(headers: HttpRequestHeaders): string {
-    return headers.eTag as string
+    return (headers.eTag as string) ?? (headers.ETag as string) ?? (headers.etag as string)
```

Comment:
Wow, would a case-insensitive search of Object.keys make sense here in case other variations such as Etag or ETAG come in?

Reply:
Probably, we can optimize and improve this going forward as we use new urls that surface other casings.